scripts/deploy-kubefed.sh

```diff
@@ -160,10 +160,6 @@ cd "$(dirname "$0")/.."
 make kubefedctl
 cd -
 
-if ! kubectl get ns "${NS}" > /dev/null 2>&1; then
```

Comment:
Does this mean that helm will create the namespace now?  I'm not clear whether this was required for helm or a vestige of the script-based deployment.

Reply:
Yes. `helm install --namespace` will create the namespace on demand
https://github.com/helm/helm/issues/4456#issuecomment-412134651